docs/section19/19-12.md

```diff
@@ -0,0 +1,39 @@
+# Section 19 - Rule 19-12         
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-12         
+**Rule Description:** The baseline system economizer high-limit shutoff shall be a dry-bulb fixed switch with set-point temperatures in accordance with the values in Table G3.1.2.7.  
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE   
+**Appendix G Section:** G3.1.2.7       
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. HVAC systems modeled with air-side economizers in the B_RMI.   
+2. Applies to buildings NOT in climate zones 0A, 0B, 1A, 1B, 2A, 3A, and 4A.   
+   
+
+**Function Calls:**  None
+## Rule Logic:   
+**Applicability Check 1 -  Check if Climate Zone Requires Economizer**  
+- Get climate zone: `climate_zone = ASHRAE229.weather.climate_zone` 
+- If climate zone is not 0A, 0B, 1A, 1B, 2A, 3A, and 4A then carry on, otherwise outcome for all HVAC systems NOT_APPLICABLE: `if climate_zone Not in ["CZ0A", "CZ0B", "CZ1A", "CZ1B", "CZ2A", "CZ3A", "CZ4A"]:`  
+    **Applicability Check 2 -  Check if HVAC System is Modeled with an Economizer**   
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Check if the system was modeled with an economizer, if not then outcome is NOT_APPLICABLE, if it is then carry on: `if hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type == "TEMPERATURE":`    
```

Comment:
I don't think we need to check for economizer type == temperature because this is already done in RDS 19-10.

Reply:
Ok, removed.